tools/crm_ticket.c

```diff
@@ -305,7 +305,7 @@ print_ticket(pcmk_ticket_t *ticket, bool raw, bool details)
                 fprintf(stdout, ", ");
             }
             fprintf(stdout, "%s=", name);
-            if (pcmk__str_any_of(name, "last-granted", "expires", NULL)) {
+            if (pcmk__str_any_of(name, PCMK_XA_LAST_GRANTED, "expires", NULL)) {
```

Comment:
I don't see "expires" set anywhere, nor documented, nor in the schemas. I made a note to self to try to track down its origin.